packages/service-library/src/servicelib/aiohttp/requests_validation.py

```diff
@@ -66,7 +71,7 @@ def handle_validation_as_http_error(
                 }
             )
 
-        raise web.HTTPBadRequest(
```

Comment:
This handles validation error from request parameters and based on the description of [400 Bad Request](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400) and [422 Unprocessable Entity](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422) I think it might be the first one.

This is ihow I understand it: reponds 400 if e.g. the client sent a parameter out of range but 422 if the password is wrong.

Reply:
then why do we get a 422 when a you miss parameters in a fastapi application? I would tend to do the same as them